frida_tools/repl.py

```diff
@@ -172,7 +173,7 @@ def _monitor_script(self):
             self._unmonitor_script()
 
             if self._user_script is not None:
-                monitor = frida.FileMonitor(self._user_script)
+                monitor = frida.FileMonitor(abspath(self._user_script))
```

Comment:
Would be good to do this in `_initialize` instead:

```Python
        def _initialize(self, parser, options, args):
            self._user_script = os.path.abspath(options.user_script)
```

(And we can use the `os` import that we already have, as the function is only used once and not in a performance-critical path.)

Reply:
That makes sense. Thanks for the update.
Will modify and update when I can!

Reply:
Applied suggested changes.